R/mod_sidebar.R

```diff
@@ -23,9 +23,9 @@ mod_sidebar_server <- function(id, r, DROPDOWNS) {
   moduleServer(id, function(input, output, session) {
     ns <- session$ns
 
-    mod_import_server("import_project_configuration_1", r, DROPDOWNS)
+    configuration_path <- mod_import_server("import_project_configuration_1", r, DROPDOWNS)
```

Comment:
`configuration_path` is already included in `r` so it is already passed to `mod_export_server`.